components/StoryCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Text, Spacer, Card, Divider, Badge, Breadcrumbs } from '@geist-ui/react';
+import { Draggable } from 'react-beautiful-dnd';
+import { Story, Owner, Label, Iteration } from '../redux/types';
+
+import Owners from './Owners';
+import Labels from './Labels';
+
+const getBorderColor = (type: string): string => {
+  if (type === 'feature') {
+    return 'gray';
+  }
+  if (type === 'bug') {
+    return 'red';
+  }
+  if (type === 'chore') {
+    return 'green';
+  }
+  if (type === 'release') {
+    return 'blue';
+  }
+  return 'gray';
+};
```

Comment:
I'd prefer to have this put in an object so:

```javascript
colors = {
  'feature': 'gray',
  'bug': 'red',
  'chore': 'green',
  'release': 'blue'
}
```
For default value of `gray`, the calling function can do `colors[type] || 'gray'`. [Pushing complexity in data structure is always better than pushing it in the code when there is a choice](https://softwareengineering.stackexchange.com/a/163231).

Reply:
Good idea, will do in a future PR